deployment/arm/global.json

```diff
@@ -87,31 +87,6 @@
                             }
                         }
                     },
```

Comment:
Should the lines for enabling AppService Auth should also be cleaned up from deploy-region script?

echo "Configuring app service authentication"
az webapp auth update \
    --name $backendAppName \
    --resource-group $resourceGroupName \
    --enabled true \
    --action AllowAnonymous \
    --twitter-consumer-key $TWITTER_KEY \
    --twitter-consumer-secret $TWITTER_SECRET \
    --allowed-external-redirect-urls "https://$frontDoorEndpoint"

Reply:
Yes, good catch.